test/scheduler/utils_test.go

```diff
@@ -248,11 +273,31 @@ func ensureCRPDeletion(crpName string) {
 
 	// Ensure that the CRP is deleted.
 	Eventually(func() error {
-		err := hubClient.Get(ctx, types.NamespacedName{Name: crpName}, &fleetv1beta1.ClusterResourcePlacement{})
+		err := hubClient.Get(ctx, types.NamespacedName{Name: crpName}, &placementv1beta1.ClusterResourcePlacement{})
 		if errors.IsNotFound(err) {
 			return nil
 		}
 
 		return err
 	}, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to delete CRP")
 }
+
+func ensureProvisionalClusterDeletion(clusterName string) {
+	// Retrieve the provisional cluster.
+	memberCluster := &clusterv1beta1.MemberCluster{}
+	Expect(hubClient.Get(ctx, types.NamespacedName{Name: clusterName}, memberCluster)).To(Succeed(), "Failed to get member cluster")
+
+	// Remove all finalizers from the provisional cluster.
+	memberCluster.Finalizers = []string{}
+	Expect(hubClient.Update(ctx, memberCluster)).To(Succeed(), "Failed to update member cluster")
```

Comment:
just wonder who will add the finalizer on membercluster

Reply:
No longer applies after the refactor.